packager/media/base/stream_info.h

```diff
@@ -49,6 +49,7 @@ enum Codec {
   kCodecDTSM,
   kCodecDTSP,
   kCodecEAC3,
+  kCodecAC4,
```

Comment:
The definition should be sorted. Can you move it after kCodecAC3?

Reply:
Sure.